worker/uniter/resolver_test.go

```diff
@@ -368,3 +368,57 @@ func (s *resolverSuite) TestRunHookStopRetryTimer(c *gc.C) {
 	c.Assert(err, gc.Equals, resolver.ErrNoOperation)
 	s.stub.CheckCallNames(c, "StartRetryHookTimer", "StopRetryHookTimer")
 }
+
+func (s *resolverSuite) TestRunsConfigChangedIfConfigHashChanges(c *gc.C) {
```

Comment:
A test for no op if hashes haven't changed?

Reply:
Ok, I'll add that.